src/main/java/com/djrapitops/extension/GriefDefenderExtension.java

```diff
@@ -95,19 +125,49 @@ public long claimedArea(UUID playerUUID) {
                 .sum();
     }
 
+    @StringProvider(
+            text = "Claim Block",
+            description = "If Eco is being used, the Blocks player can buy will be showed,\n " +
+                    "or will show like [initial + Bonus + Accrued + Reaming]",
+            iconName = "map",
+            iconColor = Color.BLUE_GREY,
+            iconFamily = Family.REGULAR,
+            showInPlayerTable = true
+    )
+    public String claimBlock(UUID playerUUID) {
+        if (api.isEconomyModeEnabled()) {
+            return "Remaining Blocks: " + getPlayerData(playerUUID).getRemainingClaimBlocks();
+        }
+        return getPlayerData(playerUUID).getInitialClaimBlocks() + "+" +
+                getPlayerData(playerUUID).getBonusClaimBlocks() + "+" +
+                getPlayerData(playerUUID).getAccruedClaimBlocks() + "+" +
+                getPlayerData(playerUUID).getRemainingClaimBlocks();
+    }
+
     @TableProvider(tableColor = Color.BLUE_GREY)
     @Tab("Claims")
     public Table claimTable(UUID playerUUID) {
         Table.Factory table = Table.builder()
-                .columnOne("Claim", Icon.called("map-marker").build())
-                .columnTwo("Area", Icon.called("map").of(Family.REGULAR).build());
+                .columnOne("Name", Icon.called("map-marker").build())
```

Comment:
You can find icon names from https://fontawesome.com/v5.15/icons?d=gallery&p=2&m=free